code/obj/item/grenades.dm

```diff
@@ -529,7 +529,55 @@ TYPEINFO(/obj/item/old_grenade/singularity)
 
 		qdel(src)
 		return
+/obj/item/old_grenade/foam_dart
+	name = "Foam dart Grenade"
+	desc = "You can make great fights with these and foam dart guns."
+	icon_state = "foam-dart"
+	det_time = 3 SECONDS
+	org_det_time = 3 SECONDS
+	alt_det_time = 6 SECONDS
+	item_state = "fragnade"
+	is_syndicate = 0
+	sound_armed = 'sound/weapons/pindrop.ogg'
+	icon_state_armed = "foam-dart1"
+	var/custom_projectile_type = /datum/projectile/bullet/foamdart/biodegradable
+	var/pellets_to_fire = 25
```

Comment:
code doc these

Reply:
how do i do that

Reply:
```
/// I'm a code doc comment!
var/foo = 1
```

Reply:
<img width="457" alt="image" src="https://user-images.githubusercontent.com/120601289/231385567-1266c615-e6b8-4d8f-85b4-4c6ff6b73a12.png">
is this what i needed to do here because i don't think i am doing it right? i am still pretty bad at coding so i won't understand everything, but if i do need to do it differently then please specify it